csrc/kernel_cache.cpp

```diff
@@ -755,6 +755,7 @@ FusionKernelRuntime* FusionExecutorCache::getKernelRuntimeFor(
     kernel_runtimes.emplace_back(std::make_unique<FusionKernelRuntime>(
         std::move(conc_fusion),
         args,
+        nullptr /* serde_buffer */,
```

Comment:
```suggestion
        /*serde_buffer=*/nullptr,
```

`/*param_name=*/` is checked by clang-tidy, so is safer.